Orange/widgets/data/owpythonscript.py

```diff
@@ -645,7 +667,7 @@ def saveScript(self):
             f.close()
 
     def initial_locals_state(self):
-        d = dict([(i.name, getattr(self, i.name, None)) for i in self.inputs])
+        d = dict([(name, getattr(self, name, None)) for name in self.inputs_names])
```

Comment:
Where is this code? I can't find `InteractiveInterpreter` (I may be blind).

If it stays a dictionary, use dict comprehension, i.e. `{name: getattr(self, name, None) for name in self.inputs_names}`.